Fixes duplicate ElementType entries.

MarinaGryphon opened the pull request:
Or, at least, an attempt.

Also, this is Ryann Foxx. I really didn't expect anyone to fix this program, wow.

MarinaGryphon committed:
Fixes duplicate ElementType entries.

Or, at least, an attempt.

Zekka commented:
Hello! This looks almost right, but I think you delete a few that only appear in the list once. Let me add a commit that puts those back.

Zekka commented:
No wait, my bad. I misread your change. Yeah, I think this is merge-able. Thanks!

Zekka committed:
Cosmetic: might as well just drop the count

Has no runtime effect -- it just makes it easier to make changes to this list later.
